examples_gbs/run_tutorial_similarity.py

```diff
@@ -0,0 +1,286 @@
+#
+# - MUTAG0: Class 1
+# - MUTAG1: Class 0
+# - MUTAG2: Class 0
+# - MUTAG3: Class 1
+
+classes = [1, 0, 0, 1]
+
+##############################################################################
+# Can we use GBS feature vectors to classify these graphs? We start by defining two-dimensional
+# feature vectors:
+
+events = [8, 10]
+max_count = 2
+
+f1 = similarity.feature_vector_sampling(m0, events, max_count)
+f2 = similarity.feature_vector_sampling(m1, events, max_count)
+f3 = similarity.feature_vector_sampling(m2, events, max_count)
+f4 = similarity.feature_vector_sampling(m3, events, max_count)
+
+import numpy as np
+
+R = np.array([f1, f2, f3, f4])
+
+print(R)
+
+##############################################################################
+# The choice of ``events`` composing the feature vectors is arbitrary and we encourage the reader
+# to explore different combinations. Note, however, that odd photon-numbered events have zero
+# probability because ideal GBS only generates and outputs pairs of photons.
+#
+# Given our points in the feature space and corresponding classifications, we can use
+# scikit-learn's `LinearSVC <https://scikit-learn.org/stable/modules/generated/sklearn.svm
+# .LinearSVC.html>`__ as our model to train:
+
+from sklearn.svm import LinearSVC
+from sklearn.preprocessing import StandardScaler
+
+R_scaled = StandardScaler().fit_transform(R)  # Transform data to zero mean and unit variance
+
+classifier = LinearSVC()
+classifier.fit(R_scaled, classes)
+
+##############################################################################
+# Here, the term "linear" refers to the *kernel* function used to calculate inner products
+# between vectors in the space. We can use a linear SVM because we have already embedded the
+# graphs in a feature space based on GBS. We can then visualize the trained SVM by plotting the
+# decision boundary with respect to the points:
+
+w = classifier.coef_[0]
+i = classifier.intercept_[0]
+
+m = -w[0] / w[1]  # finding the values for y = mx + b
+b = -i / w[1]
+
+xx = [-1, 1]
+yy = [m * x + b for x in xx]
+
+fig = plot.plot_points(R_scaled, classes)
+fig.add_trace(plotly.graph_objects.Scatter(x=xx, y=yy, mode="lines"))
+
+plotly.offline.plot(fig, filename="SVM.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/SVM.html
+#
+# This plot shows the two classes (grey points for class 0 and red points for class 1)
+# successfully separated by the linear hyperplane using the GBS feature space. Moreover,
+# recall that the two MUTAG1 and MUTAG2 graphs of class 0 are actually isomorphic. Reassuringly,
+# their corresponding feature vectors are very similar. In fact, the feature vectors of
+# isomorphic graphs should always be identical :cite:`bradler2018graph` - the small discrepancy
+# in this plot is due to the statistical approximation from sampling.

```

Comment:
It would be quite interesting to see this for different numbers of samples/ precisions!

Reply:
Thanks Maria! I think it would be a nice thing to visualise and also isn't too hard given that we already have the samples to play with.

On the other hand, I think it might not be so useful to non-experts and can take away from the main message